CMakeLists.txt

```diff
@@ -75,12 +57,39 @@ add_subdirectory(deps/wasm3 EXCLUDE_FROM_ALL)
 
 find_package(CURL REQUIRED)
 
+file(GLOB_RECURSE BUNDLE_SRC_FILES ${PROJECT_SOURCE_DIR}/src/js/*.js)
+
+set(INCLUDED_JS_FILES "eval-stdin.js;repl.js;std.js;worker-bootstrap.js")
+set(INCLUDED_JS_FILES_PATH "")
+foreach(jsfile ${INCLUDED_JS_FILES})
+    configure_file(
+        ${CMAKE_SOURCE_DIR}/src/js/${jsfile}
+        ${CMAKE_CURRENT_BINARY_DIR}/${jsfile}
+        COPYONLY)
+	list(APPEND INCLUDED_JS_FILES_PATH ${CMAKE_CURRENT_BINARY_DIR}/${jsfile})
+endforeach()
+
+add_custom_command(
```

Comment:
I appreciate the effort here, but I'd rather discuss changing the way we bundle things as a separate effort.

The main reason for how things are this way is to require virtually nothing other than a compiler on the target OS.

I am happy to re-evaluate, but not intermingled with FFI, which is already a huge change.

Can you please re-generate and commit the bundle on this PR, and open an issue so we discuss making this change afterwards? Thank you!